### DISCUSSION TITLE
Have a Look you may like one of them to work with !!

### DISCUSSION THREAD

**Discussion by Coolboyrajat** [arshsisodiya/helios-mirror#109] [Ideas]
- [ ] Multi drive Selection before uploading;
  - If have enough rights it will show automatically!!  -  I can explain what I mean to
- [ ] Uptobox can be used as Premium/Non-Premium Both;  -  I think Dawn has done this ...not sure
- [ ] ❤️‍ leech 4 gb files if provided TG_API & HASH have a Premium Account;  -  for now it checks where the user has a premium of not 
- [ ] 🤖 Reboot HEROKU form bot.
- [ ] 🥳 Surprise... Addition of a new feature.  -  there is one good thing I have in my Pocket but need suggestions and guidance

**Comment by arshsisodiya** [comment]
You can't leech 4GB files with Api and hash because telegram BOT API doesn't support 4 GB upload yet.

**Reply by Coolboyrajat** [reply]
okay if not that can "**bot be able to use one premium account to leech for other**" sorry about not being able to express my Question properly ... n what about other things that I have mentioned?

**Comment by arshsisodiya** [comment]
Uptobox thing I'm aware of, i will add later, 

And For Heroku it's better to not use Bot to change anything in heroku it increases the chances of Heroku account ban,

And Eloborate that multi drive thing and surprise feature

**Reply by Coolboyrajat** [reply]
The **concept** of Multi-Drive is the same as **[GdriveXbot](https://t.me/GdriveXbot)** but in that bot whole list of drives is shown whether you have permission or not to upload. What I'm talking about in the Multi-drive option will _only have select drives that you may be eligible to upload._ (sorting of drives depending on permission)

Here there are two possible conditions 
while setuping:
1. For Admin it shows all the eligible drives and he will sort which drive to show
      → for Eg [Open this link.](https://drive.google.com/file/d/1cBqbBoMmn3jbaKl0z-m7_bT4vMg9NNe4/view?usp=sharing) 
          → The Same thing will be asked by the user while uploading.

2. Manually select just what we do in _config.env_ by pasting multiple drives/folder ID  _"Adsbui78387", "BUIc_6dgwQsn"_
      → And if a particular Folder is selected inside the drive to upload it shows that name for Eg. let's say _Bot_Upload_ 

you can try this in the new branch ... n if you create a new also try _HEROKU reboot from bot_ give it a try 

... And for the last option to explain share with me your Telegram link not here in public discussion

**Reply by arshsisodiya** [reply]
Ok, you can find my Telegram link is 
https://t.me/nmberSEVEN
